src/lib/components/top-nav.svelte

```diff
@@ -16,6 +18,14 @@
     <slot name="left" />
   </div>
   <div class="flex items-center gap-2">
+    <NavigationButton
```

Comment:
We'll not want use this NavigationButton and instead use a regular Button

<img width="435" alt="Screenshot 2025-05-13 at 11 24 22 AM" src="https://github.com/user-attachments/assets/eabc8fab-39eb-45cf-a076-d98779665546" />